_posts/2021-10-21-segmentation-torch-android/segmentation-torch-android.Rmd

```diff
@@ -0,0 +1,279 @@
+---
+title: "Train in R, run on Android: Image segmentation with torch"
+description: > 
+ We train a model for image segmentation in R, using torch together with luz, its high-level interface. We then JIT-trace the model on example input, so as to obtain an optimized representation that can run with no R installed. Finally, we show the model being run on Android.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanasegmentationtorchandroid
+date: 10-21-2021
+bibliography: bibliography.bib
+categories:
+  - Torch
+  - Image Recognition & Image Processing
+  - R
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/segmentation_android.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+In a sense, image segmentation is not that different from image classification. It's just that instead of categorizing an image as a whole, segmentation results in a label for *every single pixel.* And as in image classification, the categories of interest depend on the task: Foreground versus background, say; different types of tissue; different types of vegetation; et cetera.
```

Comment:
```suggestion
In a sense, image segmentation is not that different from image classification. It's just that instead of categorizing an image as a whole, segmentation results in a label for *every single pixel*. And as in image classification, the categories of interest depend on the task: Foreground versus background, say; different types of tissue; different types of vegetation; et cetera.
```

Reply:
Semi-obscure rule that the punctuation doesn't get the changed font emphasis unless it's in the middle of said emphasis.